src/main/resources/templates/resource2.html

```diff
@@ -0,0 +1,225 @@
+	<div class="container">
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<p>&nbsp;</p>
+		<!--here make changes-->
+		<!--<div class="tab-content col-sm-10 dashpanel_admin">-->
+		<div class="row dashpanel col-sm-12">
+		<!--<div class="tab-content col-sm-10">-->
+		<h2 class="team-heading team-heading-current-team" th:text="#{resource.heading.current}">text heading from .properties</h2>
+
+		<ul class="topnav col-sm-2">
+			<li>
+				<a href="#">Resource Type</a>
+				<ul>
+					<li class="active"><a data-toggle="tab" href="#vul">Vulnerability Environments</a></li>
+					<li><a data-toggle="tab" href="#bigdata">Big-data Platforms</a></li>
+					<li><a data-toggle="tab" href="#blockchain">Blockchain Platforms</a></li>
+					<li><a data-toggle="tab" href="#mobile">Mobile Emulators</a></li>
+				</ul>
+			</li>
+			<li><a href="#">Network Type</a>
+				<ul>
+					<li><a href="#">Single Host</a></li>
+					<li><a href="#">Client-Server</a></li>
+					<li><a href="#">P2P network</a></li>
+					<li><a href="#">Distributed Computing</a></li>
+					<li><a href="#">Mobile</a></li>
+				</ul>
+			</li>
+			<li><a href="#">Year Created</a>
+				<ul>
+					<li><a href="#">2016</a></li>
+					<li><a href="#">2015</a></li>
+					<li><a href="#">2014</a></li>
+
+				</ul>
+			</li>
+
+		</ul>
+
+		<div class="col-sm-9">
+			<div id="vul" class="tab-panel fade in active">
+				<h3>System Vulnerability Environments</h3>
+				<div>
+					<table id="SUETable" class="table">
+					<tr>
+						<th width="1%"></th>
+						<th width="14%">Public ID</th>
+						<th width="20%">Name</th>
+						<th width="35%">Description</th>
+						<th width="30%">Action</th>
+					</tr>
+					<tr>
+						<td>1</td>
+						<td><a href="#" data-toggle="modal" data-target="#myModal">CVE-2011-2523</a></td>
+						<div class="container">
+							<div id="myModal" class="modal fade admin-modal" role="dialog">
+								<div class="modal-dialog">
+									<!--Modal Content-->
+									<div class="modal-content">
+										<div class="modal-header">
+											<button type="button" class="close" data-dismiss="modal">&times;</button>
+											<h4 class="modal-title">CVE-2011-2523</h4>
+										</div>
+										<div class="modal-body">
+											<p><b>Vulnerability</b>
+											<ul>
+												<li>CVE: 2011-2523</li>
+												<li>OSVDB: 73573</li>
+												<li>EDB: 17491</li>
+											</ul>
+											</p>
+
+											<p><b>Environment</b>
+											<ul>
+												<li>Operating system with version: Ubuntu 12.04.5 LTS</li>
+												<li> Other environment information: NIL</li>
+											</ul>
+											</p>
+
+											<p><b>Resources</b>
+											<ul>
+												<li>Link to vulnerable source: <a href="https://github.com/bb111189/BinariesForCTF" target="_blank">Click Here</a></li>
+												<li>Link to exploit: NIL</li>
+											</ul>
+											</p>
+
+
+											<p><b>Description of Vulnerability</b>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)